packages/daimo-api/src/server/cron.ts

```diff
@@ -205,4 +214,20 @@ export class Crontab {
       `Forex Transfer: ${fromDisplayName} -> ${toDisplayName} ${humanReadableValue} ${transfer.foreignToken.symbol} `
     );
   }
+
+  // Track ETH received by a user (no "from" address because currently batch receives balance difference).
+  async postRecentEthTransfer(transfer: ETHTransfer) {
+    const toName = this.nameRegistry.resolveDaimoNameForAddr(transfer.to);
+
+    if (toName == null) return;
+
+    const toDisplayName = getAccountName(
+      await this.nameRegistry.getEAccount(transfer.to)
+    );
```

Comment:
won't `toDisplayName` and `toName` always be the same? can just print `toName` i think

Reply:
Oh yea you're right

Reply:
All good, easy cleanup. I jumped the gun on merging > deployed to prod